### DISCUSSION TITLE
Sites para realizar freelancer valem a pena ?

### DISCUSSION THREAD

**Discussion by ViniciusGularte** [frontendbr/forum#1247] [Discussão]
Fala pessoal :wave: , uma pergunta para quem faz/já fez/pensa em fazer, sites  estilo Workana, Freelancer, UpWork, entre vários outros para pegar freela valem a pena ? eu sei que para quem está começando na plataforma é bem difícil( eu tentei no Freelancer.com e fui massacrado pelos indianos :joy: ).

Estes tipos de plataforma valem a pena a uma visão de longo prazo?

É possível criar uma cartela de clientes nesses sites ? 

É possível viver desses trabalhos ? 

Existem plataformas melhores ? 

Alguém do fórum contrata/já contratou pessoas dessas plataformas ? 

Conhecem pessoas que trabalham só por essas plataformas ?

**Comment by felipefialho** [comment]
Linkando: #1050

**Comment by thismarcoantonio** [comment]
A tarefa mais difícil nesse sites é pegar o próprio job, mas já me perguntaram se eu queria receber todos os trabalhos que são terceirizados numa empresa (pelo workana).

#### Respondendo suas perguntas:
- Plataformas como o workana você vai ganhando destaque, mas tem muita gente que posta e desaparece, não cheguei a ser usuário premium em nenhuma das plataformas, mas eu não boto fé a longo prazo;
- Sim, como disse ali encima, muitas pessoas pela qualidade do seu trabalho vão querer pegar seu contato, até pra pular as taxas da plataforma;
- Sim também, mas não se esqueça dos prós e contras desse cenário;
- Sobre plataformas, são essas as que conheço, mas existem alguns sistemas ai que oferecem trabalhos mais sérios (tipo a impulso), mas são totalmente voltadas pra galera mais senior;
- Nope;
- Nope;

**Comment by adhenrique** [comment]
Vale a pena, tanto para quem está iniciando como para quem já trabalha na área. 
Utilizo muito o getninjas, e sempre fecho um job ali ou aqui.

**Respondendo as questões:**

- Sim vale a pena. Ainda mais na área de tecnologia, não só pelo quesito financeiro mas principalmente se vc pensar que a longo prazo, adquirirá muito conhecimento e mercado.
- Sim. Hoje tenho clientes que preferem me contatar diretamente.
- Sim. 
- Eu recomendaria o Getninjas - apesar de ter uma taxa um pouco alta - vale a pena. Procure saber.
- Não.
- Não.

**Comment by pietrobelluno** [comment]
@adhenrique opa, meio offtopic mas, como funciona esse Getninjas? Eu me cadastrei e não entendi se eu tenho que comprar moedas pra receber pedidos ou se é só esperar?

**Comment by ViniciusGularte** [comment]
@pietrobp  pelo que eu entendi( me corrijam se eu estiver errado) você compra um pacote de moedas, você consegue ver os serviços mas para liberar o contato da pessoa você tem que pagar com as moedas.

**Comment by matheus-rosin** [comment]
@ViniciusGularte exatamente isso... e o legal é que às vezes aparecem uns trampos bem "cavalos", não é só bico.

Mas tem um detalhe: até três freelancers podem abrir o contato do mesmo serviço, então abrir o contato (o que significa gastar um pouco das moedinhas que precisa comprar) não garante o serviço. O segredo é:

- não querer abraçar o mundo inteiro, isto é, selecionar bem quais serviços abrir;
- tentar escolher serviços que ninguém viu o contato ainda;
- assim que abrir o contato, tentar falar com a pessoa o mais rápido possível, e ser bem solícito, já oferecendo um bom serviço, dando ideias, mostrando interesse etc.

Com o tempo se pega a manha rs e no Getninjas acontece também de a pessoa pegar seu contato e continuar pedindo outros serviços diretamente para você.

A Impulso sei que é muito boa, mas ainda não cheguei a trabalhar com eles porque precisa de uns requisitos (no caso, não tenho tempo suficiente de mercado ainda). Parecido com a Impulso é a Geekhunter (pelo menos acho), mas também precisa de, no mínimo, 3 anos de experiência profissional.

**Comment by brunodesde1987** [comment]
@ViniciusGularte acredito que depende do site mas é possível sim. Fiquei 7 meses trabalhando na plataforma 99freelas e através dela consegui o emprego atual.

Os problemas mais comuns são os citados pelos colega acima. Dá um certo trabalho pois muita gente posta em várias plataformas e se fecham em alguma delas, eles não voltam para falar com você, simplesmente somem. Então a porcentagem de propostas enviadas vs propostas fechadas é bem desproporcional. 

No começo também é difícil pois a maioria das plataformas possuem algum esquema de pontuação ou plano premium, o que traz mais confiança para quem está contratando mas ao mesmo tempo dificulta para o novato na plataforma.

O que recomendo para contornar esse problema é oferecer uma prévia do serviço gratuitamente para que o cliente conheça seu trabalho antes de se comprometer financeiramente. É claro que isso tem que ser feito com cautela mas pra mim funcionou muito e depois que ganhei uma certa reputação isso não foi mais necessário.

No caso do 99freelas acho que compensa pagar o premium pois é relativamente barato e fornece um pouco mais de competitividade. Boa sorte!

**Comment by edimilson2016** [comment]
agora surgiu uma nova plataforma em são paulo , moro em são paulo pelo menos uma né  se quiserem conhecer ok.http://quarentaconto.com/
